maxfw/utils/image_functions.py

```diff
@@ -23,7 +23,7 @@ def _is_numpy_image(img):
     return isinstance(img, np.ndarray) and (img.ndim in {2, 3})
 
 
-def to_pil_image(pic, mode=None):
+def to_pil_image(pic, target_mode, mode=None):
```

Comment:
Added a `target_mode` attribute to specify the format of the image after loading it in memory. The original `mode` attribute specifies how the bytes should be interpreted, the `target_mode` converts the mode after reading the file in.